components/engine_rocks/src/db.rs

```diff
@@ -68,11 +75,17 @@ impl AsMut<Rocks> for Arc<DB> {
     }
 }
 
+impl Into<Arc<DB>> for Rocks {
```

Comment:
Could we `impl From<Rocks> for Arc<DB>`  instead?

Reply:
This is unused and I was going to delete it anyway, so it is deleted now.